include/exadg/time_integration/restart.h

```diff
@@ -27,13 +27,35 @@
 #include <boost/archive/binary_oarchive.hpp>
 #include <boost/archive/text_iarchive.hpp>
 #include <boost/archive/text_oarchive.hpp>
+#include <algorithm>
 #include <fstream>
 #include <sstream>
 
 // deal.II
 #include <deal.II/base/mpi.h>
+#include <deal.II/distributed/fully_distributed_tria.h>
+#include <deal.II/distributed/solution_transfer.h>
+#include <deal.II/distributed/tria.h>
+#include <deal.II/grid/grid_generator.h>
+#include <deal.II/grid/manifold.h>
+#include <deal.II/grid/tria.h>
 #include <deal.II/lac/la_parallel_block_vector.h>
 #include <deal.II/lac/la_parallel_vector.h>
+#include <deal.II/lac/solver_cg.h>
+#include <deal.II/matrix_free/fe_point_evaluation.h>
+#include <deal.II/numerics/vector_tools.h>
+#include <deal.II/particles/data_out.h>
+#include <deal.II/particles/particle_handler.h>
```

Comment:
Do you really need these two include files?

Reply:
no, that is probably leftover from a previous version of this PR. Thanks for spotting it!